man/systemd.resource-control.xml

```diff
@@ -228,9 +228,11 @@
           reclaimed as long as memory can be reclaimed from unprotected units.</para>
 
           <para>Takes a memory size in bytes. If the value is suffixed with K, M, G or T, the specified memory size is
-          parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively. This controls the
-          <literal>memory.low</literal> control group attribute. For details about this control group attribute, see
-          <ulink url="https://www.kernel.org/doc/Documentation/cgroup-v2.txt">cgroup-v2.txt</ulink>.</para>
+          parsed as Kilobytes, Megabytes, Gigabytes, or Terabytes (with the base 1024), respectively. Alternatively, a
+          percentage value may be specified, which is taken relative to the installed physical memory on the
```

Comment:
Hmm, but "available memory" sounds like "free memory" to me, i.e. not the total memory assigned to the system, but just the stuff that isn't used. I actually had it with "available" first, but then changed it to "installed", because i figured that would be less confusing...

I see your point though, but I am not sure "available" is the best wording, can you think of something else?